### PR TITLE
removing facebook login, adding username

### PR DESCRIPTION
**Pull request by dconroy**
Adds username as primary identifier. Also allows for people to login with verified email.

### CODE REVIEW

**Comment by dconroy** [comment]
Review this PR 2nd

File: src/config.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ export default {
     },
     cognito: {
       REGION: "us-east-1",
-      USER_POOL_ID: "us-east-1_OvHRINlq8",
-      APP_CLIENT_ID: "7poun2fmqke6jtsq6er8eo4ab7",
+      USER_POOL_ID: "us-east-1_FJaKcw4bh",
+      APP_CLIENT_ID: "khbkequqcckarsm3j0ra4tu4p",
```
Comment (johnosullivan) [inline src/config.js]: Good!

File: src/index.js [hunk 2]
```diff
@@ -15,6 +15,9 @@ Amplify.configure({
       identityPoolId: config.cognito.IDENTITY_POOL_ID,
       userPoolWebClientId: config.cognito.APP_CLIENT_ID
     },
+    Analytics: {
+      disabled: true
```
Comment (johnosullivan) [inline src/index.js]: Why disabled?

Comment (dconroy) [reply src/index.js]: hushes an error message

**Pull request merged**